### ISSUE TITLE
Unexected error upon clicking Config

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/profileview#78]
Installed the (31Aug2021) ZIP download into 5.1.4 64bit on Windows.
loaded my Tree & switched to Relationships View, Profile mode
clicked Config and got the following error.
  thought that an old .ini file might be the problem. So exited and cleaned out the 3 .ini files for Relationships_profileview and repeated.  Same error.

```
43709: ERROR: grampsapp.py: line 174: Unhandled exception
Traceback (most recent call last):
  File "C:\Program Files\GrampsAIO64-5.1.4\gramps\gui\viewmanager.py", line 1291, in config_view
    self.active_page.configure()
  File "C:\Program Files\GrampsAIO64-5.1.4\gramps\gui\views\pageview.py", line 589, in configure
    'view': self.get_title()})
  File "C:\Program Files\GrampsAIO64-5.1.4\gramps\gui\views\pageview.py", line 603, in __init__
    dialogtitle=dialogtitle, on_close=on_close)
  File "C:\Program Files\GrampsAIO64-5.1.4\gramps\gui\configure.py", line 196, in __init__
    self.__setup_pages(configure_page_funcs)
  File "C:\Program Files\GrampsAIO64-5.1.4\gramps\gui\configure.py", line 209, in __setup_pages
    labeltitle, widget = func(self)
  File "C:\Users\DistrictSupport\AppData\Roaming\gramps\gramps51\plugins\profileview-master\view\pages\page_base.py", line 314, in global_panel
    self.dbstate, self.uistate, None, self.config, None, None
TypeError: __init__() takes 6 positional arguments but 7 were given
```

**Comment by cdhorn** [comment]
I should have tested that, sorry, missed it in refactoring the callbacks. Fixed in ed4bf824d554aa943e0752469a18deebe74bca89

**Comment by emyoulation** [comment]
Thank you!
